waku/waku_rendezvous/protocol.nim

```diff
@@ -0,0 +1,181 @@
+{.push raises: [].}
+
+import
+  std/[sugar, options],
+  results,
+  chronos,
+  chronicles,
+  metrics,
+  libp2p/protocols/rendezvous,
+  libp2p/switch,
+  libp2p/utility
+
+import
+  ../node/peer_manager,
+  ../common/enr,
+  ../waku_enr/capabilities,
+  ../waku_enr/sharding,
+  ../waku_core/peers,
+  ../waku_core/topics,
+  ./common
+
+logScope:
+  topics = "waku rendez vous"
```

Comment:
```suggestion
  topics = "waku rendezvous"
```

I would personally write "rendezvous" everywhere instead of "rendez vous" for consistency (it will also make everything easier when performing searches)

Reply:
Yes, sorry for that in French it's 2 words not one...